src/utils/apollo.types.ts

```diff
@@ -109,18 +109,33 @@ export interface product {
 
 // Query returns
 export interface getListOfProvidersReturnTypes {
-  providers: provider[];
+  providers: Provider[];
   providers_aggregate: query_aggregate;
 }
 
 export interface getProviderReturnType {
-  providers: provider[];
+  providers: Provider[];
 }
 
 export interface getProductByIdReturnTypes {
   products: product[];
 }
 
+// Events
+export interface Event {
+  __typename: 'events';
+  id: string;
+  title: string;
+  image_url?: string;
+  date: string;
+  type: 'PRIVATE' | 'PUBLIC';
```

Comment:
I am thinking in change types to 

`ATTENDANCE | ANNOUNCE`

- ATTENDANCE: It's a private event that a member has to inscribe to.
- ANNOUNCE: It's a public event with some requisites to participate but you have to send a form and/or follow instructions.

What do you think?

Reply:
I like `ATTENDANCE`, but I don't know if `WORKSHOP` is the best word

Reply:
I like both. 

"workshop" is weird to me since I didn't know the word, but I've checked and it's equivalent to "Taller" in Spanish, remember me when I meet at the school with a group of people to learn/do a new thing between all.

If exist a better word could be considered

Reply:
BTW: I think I should change this:

```
   type: 'PRIVATE' | 'PUBLIC'; 
```

for a "enum" or something similar

Reply:
Thanks, @anaceciliamolinares so that we will use WORKSHOP